src/index.js

```diff
@@ -1,34 +1,23 @@
 import './style.css';
+import { addTodo, removeTodo, completeTodo } from './modules/addTodo';
+import todo from './modules/todo';
 
-class Task {
-  constructor(id, description, completed) {
-    this.id = id;
-    this.description = description;
-    this.completed = completed;
-  }
-}
+const form = document.getElementById('form');
+const checkbox = document.getElementById('checkbox');
```

Comment:
- [ ] You are trying to select an element by ID checkbox, but there is no such id on the checkbox. As a result the application is crushing. I would recommend to select elements by className and iterate through all of them:
```

Array.from(checkbox).forEach((checkbox) => {
    checkbox.addEventListener('change', removeTodo);
    checkbox.addEventListener('change', completeTodo);
  });
```

- [ ] Please add `removeTodo` function on `fas fa-trash-alt` icon click instead of checkbox icon click 😉. Currently tasks are not being deleted:

https://user-images.githubusercontent.com/77038610/151672504-89c73fd4-47a0-4074-afaf-d700511b24fe.mp4

Reply:
Regarding this point, the trash can triggers always the input function and tries to submit what it is on the input text field. It is a button, and it has no submit type. I think the better workaround is to do it a span.